openbb_platform/platform/provider/openbb_provider/standard_models/index_info.py

```diff
@@ -0,0 +1,40 @@
+"""Index Info  data model."""
+
+from typing import List, Optional, Set, Union
+
+from pydantic import Field, field_validator
+
+from openbb_provider.abstract.data import Data
+from openbb_provider.abstract.query_params import QueryParams
+
+
+class IndexInfoQueryParams(QueryParams):
+    """Index Info Query Params"""
+
+    symbol: str = Field(description="The index ticker symbol.")
+
+    @field_validator("symbol", pre=True, check_fields=False, always=True)
+    def upper_symbol(cls, v: Union[str, List[str], Set[str]]):
+        """Convert symbol to uppercase."""
+        if isinstance(v, str):
+            return v.upper()
+        return ",".join([symbol.upper() for symbol in list(v)])
+
+
+class IndexInfoData(Data):
+    """Index Info Data."""
+
+    symbol: str = Field(description="The index ticker symbol.")
+    name: str = Field(description="The name of the index.")
+    description: Optional[str] = Field(
+        description="The short description of the index.", default=None
+    )
+    methodology: Optional[str] = Field(
+        description="URL to the methodology document.", default=None
+    )
+    factsheet: Optional[str] = Field(
+        description="URL to the factsheet document.", default=None
+    )
+    nb_constituents: Optional[int] = Field(
```

Comment:
Can we either explain what nb_ means in the description or just use constituents?

Reply:
We could make this, `num_constituents` if that is more clear?

Reply:
yes this is way better